nni/retiarii/nn/pytorch/nn.py

```diff
@@ -4,30 +4,36 @@
 import torch.nn as nn
 from typing import (Any, Tuple, List, Optional)
 
+from ...utils import add_record
+
 _logger = logging.getLogger(__name__)
 _logger.setLevel(logging.INFO)
 
-_records = None
-
-def enable_record_args():
-    global _records
-    _records = {}
-    _logger.info('args recording enabled')
-
-def disable_record_args():
-    global _records
-    _records = None
-    _logger.info('args recording disabled')
-
-def get_records():
-    global _records
-    return _records
-
-def add_record(name, value):
-    global _records
-    if _records is not None:
-        assert name not in _records, '{} already in _records'.format(name)
-        _records[name] = value
+__all__ = [
```

Comment:
Is this  `nn.__all__` plus a few others?

Reply:
yes, ours, for example placeholder